nuxt.config.js

```diff
@@ -190,6 +190,13 @@ export default {
         file: 'id-ID.js',
         name: 'Indonesian',
         domain: 'https://id.nuxtjs.org'
+      },
+      {
+        code: 'tr',
```

Comment:
can we comment out this for now like the spanish version as adding this will add a link to the lang selector in footer and we don't have a repo set up for turkish yet therefore this link will break. we are working on improving this but for now we need to hide this until we have a solution. thanks